src/ethereum_test_tools/rpc/rpc.py

```diff
@@ -0,0 +1,118 @@
+"""
+JSON-RPC methods and helper functions for EEST consume based hive simulators.
+"""
+
+import time
+from typing import Dict, List, Literal, Union
+
+import requests
+from jwt import encode
+from tenacity import retry, stop_after_attempt, wait_exponential
+
+from ethereum_test_tools import Address
+
+
+class BaseRPC:
+    """
+    Represents a base RPC class for every RPC call used within EEST based hive simulators.
+    """
+
+    def __init__(self, client):
+        self.client = client
+        self.url = f"http://{client.ip}:8551"
+        self.jwt_secret = (
+            b"secretsecretsecretsecretsecretse"  # oh wow, guess its not a secret anymore
```

Comment:
Us this `jwt_secret` value used by Hive? Or any reason for this default?

Reply:
Its required for the engine rpc commands not the eth rpc commands! 

I can remove it and and add it with then engine PR?

Reply:
I removed the jwt auth. here: [32bfef2](https://github.com/ethereum/execution-spec-tests/pull/556/commits/32bfef2774a96fe6f7754abd9d29170b9dee16ee)

As its only specific to engine calls I can re-add it to a seperate PR.

Reply:
You can leave it :) I just thought we could change the comment to say whey we use this default.